### ISSUE TITLE
My Aqara B1 curtain motor (ZNCLDJ12LM) have some issues

### ISSUE THREAD

**Issue by jcob9320** [Koenkk/zigbee2mqtt#17807]
### What happened?

I'm using two B1 motors (ZNCLDJ12LM). They each have one different problem.

1. First motor 
   When I open or close the curtain in my HA system or soft touch, motors does not fully close or open, leaving 1% remaining. So, it either shows ‘99% open’ or ‘1% open’.

![12](https://github.com/Koenkk/zigbee2mqtt/assets/129671069/befdb0ec-1d9b-4b3a-8178-633fa39fa0b5)
![13](https://github.com/Koenkk/zigbee2mqtt/assets/129671069/f03a6562-d567-4e84-89e5-f1b5b840a479)
("열림" =  means open)

2. Second motor
   The second motor opens and closes perfectly. But the “running”, "motor_state" of the motor is not displayed when it is in operation. 
When I run the motor is in the process of opening, but it shows "running=false" and "motor_state=stop” in the z2m web UI.
It is inconvenient because I cannot monitor the active state of the motor in real-time on the z2m web UI.

![14](https://github.com/Koenkk/zigbee2mqtt/assets/129671069/bf68c27e-aa1d-4684-b19e-400e6cbd9c35)

The first motor's firmware version is 2019.www / 20200803
The second motor's firmware version is 2019.www / 20190329
I try to OTA firmware update for first motor (second motor does not exist on OTA) but I failed. 
(error shows= Failed to check update available for 'first motor' (Failed to find endpoint which support OTA cluster)

I recalibration, and repaired several times.

### What did you expect to happen?

Visually see the curtain opening or closing as it is being opened or closed.
The curtain to be displayed as 100% open or 0% closed when it is fully opened or closed.

### Zigbee2MQTT version

1.30.4

### Adapter firmware version

20221226

### Adapter

SONOFF DONGLE-P (cc2652)

**Comment by JoooostB** [comment]
Did you manage to fix your issues @jcob9320?

**Comment by jcob9320** [comment]
Hi @JoooostB. Apologize for my late reply. I checked your reply too late. In summary, I managed to update the firmware for both motors connected to the Aqara M2 Hub. One motor's 'motor state' issue has been resolved (although the 1%/99% problem remains unsolved). But the other motor still has unresolved issues with 'motor state' and 'state.' Hence, I decided to give up and created a template cover sensor to use instead.
